doc/user/content/releases/v0.48.md

```diff
@@ -1,13 +1,71 @@
 ---
 title: "Materialize v0.48"
 date: 2023-03-29
-released: false
+released: true
+patch: 2
 ---
 
-{{< warning >}}
-This version of Materialize is not yet released.
-{{< /warning >}}
-
 ## v0.48.0
 
-* No documented changes yet.
+#### SQL
+
+* Introduce **object owners**, who can manage privileges for other roles on
+  each object in the system by adding or revoking grants. In this release,
+  object owners have limited functionality and are assigned as follows:
+
+  * All objects that exist at the time of a new Materialize deployment
```

Comment:
nit: Since each rule stands on its own, I'd end the first 2 bullets with a period rather than a semicolon.